cinnamon/src/dark/dark-assets/checkbox/checkbox-checked-focused-.svg

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
please change the `checkbox-*` color to yaru's orange-color rather than using an arbitrary color outside the palette

Reply:
I notice all the assets provided in this PR have the same issue, please modify and change the color to yaru's-orange and make sure to minimze the SVG size as well.

Reply:
No we recently dropped purple, you need to use the ubuntu orange everywhere.

Reply:
It would certainly be easier for you to copy the ones `checboxes` and `switches` svgs present under `gnome-shell`, paste and rename them accordingly :)